src/engine/instrument.h

```diff
@@ -94,6 +94,7 @@ enum DivInstrumentType: unsigned short {
   DIV_INS_GBA_MINMOD=61,
   DIV_INS_BIFURCATOR=62,
   DIV_INS_SID2=63, // coincidence!
+  DIV_INS_OPL4_PCM=64,
```

Comment:
Please don't. Use the MultiPCM type.

Reply:
Someone on Discord might've said that the OPL4 PCM is different from the GEW8 PCM